site/en/docs/extensions/mv3/manifest/index.md

```diff
@@ -67,7 +67,6 @@ discusses each field.
   <span class="token property">"<a href="/docs/extensions/mv3/manifest/nacl_modules">nacl_modules</a>"</span><span class="token operator">:</span> <span class="token punctuation">[</span>...<span class="token punctuation">]</span><span class="token punctuation">,</span>
   <span class="token property">"natively_connectable"</span><span class="token operator">:</span> ...<span class="token punctuation">,</span>
   <span class="token property">"oauth2"</span><span class="token operator">:</span> ...<span class="token punctuation">,</span>
-  <span class="token property">"<a href="/docs/extensions/mv3/manifest/offline_enabled">offline_enabled</a>"</span><span class="token operator">:</span> <span class="token boolean">true</span><span class="token punctuation">,</span>
```

Comment:
I thought extensions could also mark themselves as offline enabled. I'd like to follow up on this offline just to make sure I'm on the same page.

Reply:
Yep, they can (though this is only consumed by the webstore).  We should probably leave it in